### DISCUSSION TITLE
Welcome to OpticSim.jl Discussions!

### DISCUSSION THREAD

**Discussion by brianguenter** [brianguenter/OpticSim.jl#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fortunewalla** [comment]
Do you have a roadmap in mind? 

Could you publish a tentative one so that those interested would understand what is the general direction planned for the project? Thanks.

**Comment by brianguenter** [comment]
As you may have noticed the project has been transitioned from the Microsoft open source website to my personal website. There used to be three people maintaining the project but now it's just me. Updates and bug fixes will go more slowly than before. I don't have a long term plan for OpticSim yet. But over the next few months I will have more time to spend thinking about this.

**Reply by fortunewalla** [reply]
Sure. I understand. The intended scale of this project would definitely need some time for thought. All the best. 🙏
